### DISCUSSION TITLE
Questions about 20221222 Aquila custom compile

### DISCUSSION THREAD

**Discussion by Crazy-Charles** [mriscoc/Ender3V2S1#577] [General]
I have just compiled the new version of the firmware for my Aquila X2 which has a modified 4.2.7 board and Sprite Pro.

1) - The proUI toolbar has dropped the "T" from the Tramming Wizard text so I now have a "ramming Wizard" 

2) - MPC - auto tune, the first step it does is to reduce the temp to ambient (22C?) by switching on the hotend fan BUT i have modified my board to use E0_AUTO_FAN_PIN PC1 so the fan only starts/stops at 50C the same as my strandard E3S1 using the V24S1_V301 board, so the only fan MPC switched on was the case cooling fan!

3) - BED PID auto tune, this doesn't display the target temp, you can see the actual temp rising but target remains at zero

**Comment by Crazy-Charles** [comment]
Scrub the MPC part, it was me being a dork! It uses the part cooling fan to bring the nozzle temp down to bed temp before starting.

The Tramming Wizard text drops the T from the toolbar and the toolbar setup menu.

Another thing i have noticed is that the bed remains heated after the auto mesh has been completed?

**Comment by mriscoc** [comment]
1. In my test the "T" for Tramming Wizard is there. So, check your configuration files, make a change at time, compile and verify.
2. Normal MPC behavior
3. PID screen only shows the target temperature in the main area of the screen, not in the dashboard.
![image](https://user-images.githubusercontent.com/2745567/209577022-192f20d9-5fe3-48b9-9a48-834a4cbbb9b3.png)

**Reply by Crazy-Charles** [reply]
Update - 
Compiled for the Ender3S1-FI and no problems - Tramming is shown correctly
Compiled using the Ender3V2-427-BLTUBL-T13 files from Special Configurations with NO changes and problem is present.
Searching through the binary files for plain text it doesn't appear to be a spelling issue more of an alignment issue?


![20221227_112526](https://user-images.githubusercontent.com/119892416/209660992-05a4454a-73bc-47e5-b094-d8c243f43250.jpg)
![20221227_112507](https://user-images.githubusercontent.com/119892416/209660998-fa71827e-6267-4741-9510-2fa141abcfca.jpg)

**Reply by mriscoc** [reply]
> Compiled using the Ender3V2-427-BLTUBL-T13 files from Special Configurations with NO changes and problem is present.

Without changes the problem shouldn't be there. I can't replicate the issue.

**Comment by Crazy-Charles** [comment]
Reason found.
I had modded my board to drive an LED light using #define CASE_LIGHT_ENABLE and also enabled #define CASE_LIGHT_MENU, it worked ok but caused the issue with the missing "T", disabling it and the text is back to normal.
